ocamldoc/Makefile

```diff
@@ -403,15 +403,13 @@ stdlib_man/Pervasives.3o: $(OCAMLDOC) $(STDLIB_MLIS)
 	$(MAKE) unprefix_stdlib_for_ocamldoc
 	$(MKDIR) stdlib_man
 	$(OCAMLDOC_RUN) -man -d stdlib_man -nostdlib -I stdlib_non_prefixed \
-	-t "OCaml library" -man-mini $(STDLIB_MLIS) \
-	-initially-opened-module Pervasives
+	-t "OCaml library" -man-mini $(STDLIB_MLIS)
 
 stdlib_html/Pervasives.html: $(OCAMLDOC) $(STDLIB_MLIS)
```

Comment:
I am completely ignorant of how the "ocamldoc hack" works so can't venture a reason, but this is what the HTML doc for `Pervasives` looks in my machine (obtained by `make stdlib_html/Pervasives.html`):
<img width="577" alt="screen shot 2018-08-21 at 21 22 50" src="https://user-images.githubusercontent.com/113560/44413211-6a28e380-a588-11e8-9b05-e5ba3a88704a.png">

Reply:
I will have a look but I believe that it is easy enough to fix this issue by adding a basic support for `include module type of struct include ocamldoc end` in ocamldoc .

Reply:
thanks

Reply:
Could you rebase now that #2000 has been merged?